hs_core/models.py

```diff
@@ -1185,9 +1186,8 @@ def bag_url(cls, resource_id):
     @classmethod
     def scimeta_url(cls, resource_id):
         scimeta_path = "{resource_id}/data/resourcemetadata.xml".format(resource_id=resource_id)
-        istorage = IrodsStorage()
-        scimeta_url = istorage.url(scimeta_path)
-
+        scimeta_url = reverse('rest_download',
+                          kwargs={'path': scimeta_path})
```

Comment:
@jbradberry Thanks for the useful info. Have taken this suggestion and removed a few more apps.py files from setup.cfg to put them under pylint check as well with this comment added to receivers import line ignoring that warning on import not being used.